jsapp/js/account/plan.component.tsx

```diff
@@ -0,0 +1,403 @@
+            aria-label={'Toggle to month options'}
+            onChange={() => dispatch({type: 'month'})}
+            checked={state.filterToggle}
+          />
+          <label htmlFor='switch_right'> {t('Monthly')}</label>
+        </form>
+        <div
+          className={styles.currentPlan}
+          style={{
+            gridRow: 0,
+            gridColumn: 1 + filterPrices().findIndex(isSubscribedProduct),
+            display:
+              filterPrices().findIndex(isSubscribedProduct) >= 0 ? '' : 'none',
+          }}
+        >
+          {t('your plan')}
+        </div>
+        {filterPrices().map((price: Price, i: number) => (
+          <div className={styles.planContainer} key={i}>
+            <h1 className={styles.priceName}> {price.name} </h1>
+            <div className={styles.priceTitle}>
+              {typeof price.prices.human_readable_price === 'string' &&
+                (price.prices.human_readable_price.includes('$0.00')
+                  ? t('Free')
+                  : price.prices.human_readable_price)}
+            </div>
+
+            <ul>
+              {Object.keys(price.metadata).map(
+                (featureItem: string) =>
+                  featureItem.includes('feature_list_') && (
+                    <li key={featureItem}>
+                      <div className={styles.iconContainer}>
+                        <Icon
+                          name='check'
+                          size='m'
+                          classNames={
+                            price.name === 'Professional plan'
+                              ? [styles.tealCheck]
+                              : [styles.stormCheck]
+                          }
+                        />
+                      </div>
+                      {price.metadata[featureItem]}
+                    </li>
+                  )
+              )}
+            </ul>
+
+            {!isSubscribedProduct(price) && (
+              <Button
+                type='full'
+                color='blue'
+                size='m'
+                label={t('Upgrade')}
+                onClick={() => upgradePlan(price.prices.id)}
+                aria-label={`upgrade to ${price.name}`}
+                aria-disabled={buttonsDisabled}
+                isDisabled={buttonsDisabled}
+              />
+            )}
+            {isSubscribedProduct(price) &&
+              state.organization?.uid &&
+              price.name !== 'Community plan' && (
+                <Button
+                  type='full'
+                  color='blue'
+                  size='m'
+                  label={t('Manage')}
+                  onClick={managePlan}
+                  aria-label={`manage your ${price.name} subscription`}
+                  aria-disabled={buttonsDisabled}
+                  isDisabled={buttonsDisabled}
+                />
+              )}
+            {price.name === 'Community plan' && (
+              <div className={styles.btnSpacePlaceholder} />
```

Comment:
Why do we need this? Now it is being rendered unnecessarily here:

<img width="471" alt="Screenshot 2023-04-03 at 12 01 50" src="https://user-images.githubusercontent.com/2521888/229494764-06d7a93b-63af-45d3-bb68-17ff34caf262.png">

Reply:
So the purpose was the create space when there is no button - I am a little confused why a button is showing there